docs/framework/toc.yml

```diff
@@ -331,8 +331,6 @@ items:
   items:
   - name: Application domains and assemblies
     href: app-domains/
-  - name: Resources in desktop apps
-    href: resources/
```

Comment:
Should we keep this node but just point it to the new location?

Reply:
I thought that was not advised, does that end up duplicating TOC bits? I'm not sure.

Reply:
Okay, probably better to leave as-is then.